app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CommentsController < ApplicationController
+    def create
+      @article = Article.find(params[:article_id])
+      @comment = @article.comments.create(comment_params)
+      redirect_to article_path(@article)
+    end
+  
+    private
+      def comment_params
+        params.require(:comment).permit(:commenter, :body)
```

Comment:
What does require do?

Reply:
Require is a method that guarantees that the comment parameter is present, otherwise, it generates an error.